packages/e2e/cypress.json

```diff
@@ -1,10 +0,0 @@
-{
-  "baseUrl": "http://localhost:3000/",
-  "defaultCommandTimeout": 15000,
-  "videoUploadOnPasses": false,
-  "env": {
-    "TAGS": "not @skip"
```

Comment:
Is `not @skip` something we would still need?

I thought this helped us skip tests when we do `yarn e2e dev`.

Reply:
The library has smart tagging and always skips `@skip` tags now: https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/docs/tags.md#smart-tagging